### PR TITLE
feat(PDEV-2231: Create a v10 route that returns a list of clients for a firm)

### PR DESCRIPTION
**Pull request by kleekich**

### CODE REVIEW

File: api/v10/examples/getClientAccountList.res.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "email" : "test@example.com",
+    "guid" : "123ABC"
```
Comment (kongmalin) [inline api/v10/examples/getClientAccountList.res.json]: Can put dummy guid string like 162f5e2a-ab8d-4572-95f8-90b62e8b2742

Comment (kleekich) [reply api/v10/examples/getClientAccountList.res.json]: sure!

File: api/v10/index.raml [hunk 2]
```diff
@@ -49,6 +49,25 @@
               Can be one of the following reasons:
 
               invalid firmGuid, invalid email address
+    post:
```
Comment (jacobsung) [inline api/v10/index.raml]: It's missing your route info between line 51 and 52.

**Comment by jacobsung** [comment]
You should build and launch api-doc-consumer locally to your browser, and see all info are coming thru.

File: api/v10/index.raml [hunk 3]
```diff
@@ -32,6 +32,25 @@
...
+              schema: !include schema/getClientAccountList.res.json
+              example: !include examples/getClientAccountList.res.json
+        204:
+          description: |
+            Could not find any accounts that matches email address
+        400:
+          description: |
+            Can be one of the following reasons:
+            
+            invalid firmGuid
```
Comment (jacobsung) [inline api/v10/index.raml]: Question: After looking at clientAccountService.getClientAccountsOfEmailLike, I don't see a way to get a 400 from an invalid firmGuid.

Comment (kleekich) [reply api/v10/index.raml]: Yes, you are correct. Thank you for pointing that out. I need to update with http status codes with what my responses returning. I will take a closer look with api-docs for formats and details. Thank you.

**Pull request merged**